### ISSUE TITLE
Missing `list` symbols in the object inventory

### ISSUE THREAD

**Issue by fc6092dd-1f2c-432f-a95c-2090f67d6d46** [python/cpython#89923]
BPO | [45762](https://bugs.python.org/issue45762)
--- | :---
Nosy | @ezio-melotti, @merwok, @willingc, @JulienPalard, @Akarys42

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2021-11-09.08:18:16.882>
labels = ['3.11', 'docs']
title = 'Missing `list` symbols in the object inventory'
updated_at = <Date 2021-11-13.19:55:47.322>
user = 'https://github.com/Akarys42'
```

bugs.python.org fields:
```python
activity = <Date 2021-11-13.19:55:47.322>
actor = 'eric.araujo'
assignee = 'docs@python'
closed = False
closed_date = None
closer = None
components = ['Documentation']
creation = <Date 2021-11-09.08:18:16.882>
creator = 'Akarys'
dependencies = []
files = []
hgrepos = []
issue_num = 45762
keywords = []
message_count = 2.0
messages = ['406004', '406289']
nosy_count = 6.0
nosy_names = ['ezio.melotti', 'eric.araujo', 'docs@python', 'willingc', 'mdk', 'Akarys']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue45762'
versions = ['Python 3.11']
```

</p></details>

**Comment by StanFromIreland** [comment]
Currently, I only see these in the index:

> [append() (array.array method)](https://docs.python.org/3/library/array.html#array.array.append)
>   -  [(collections.deque method)](https://docs.python.org/3/library/collections.html#collections.deque.append)
>   -  [(email.header.Header method)](https://docs.python.org/3/library/email.header.html#email.header.Header.append)
>   -  [(imaplib.IMAP4 method)](https://docs.python.org/3/library/imaplib.html#imaplib.IMAP4.append)
>   -  [(sequence method)](https://docs.python.org/3/library/stdtypes.html#index-24)
>   -  [(xml.etree.ElementTree.Element method)](https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element.append)
> 
> [append_history_file() (in module readline)](https://docs.python.org/3/library/readline.html#readline.append_history_file)
> [appendChild() (xml.dom.Node method)](https://docs.python.org/3/library/xml.dom.html#xml.dom.Node.appendChild)
> [appendleft() (collections.deque method)](https://docs.python.org/3/library/collections.html#collections.deque.appendleft)

**Comment by merwok** [comment]
Yes, as said before, there’s only mention of sequence, not list.
See my previous message for analysis and ideas and share your opinion about what we should do!

**Comment by serhiy-storchaka** [comment]
We can simply add the index entries for each standard type:
```rst
.. index::
   single: append() (sequence method)
   single: append() (list method)
   single: append() (bytearray method)
   single: append() (collections.deque method)
```
Or only for builtin types.
